docs/modules/servers/pages/local/conf/index.adoc

```diff
@@ -0,0 +1,23 @@
+= James Local Mail Server Configuration
+:navtitle: Configuration
+
+ * Domain names (domainlist.xml)
+ * Database (james-database.properties)
+ * LMTP (lmtpserver.xml)
+ * POP3 (pop3server.xml)
+ * DNS (dnsservice.xml)
+ * Loggin (logback.xml)
+ * ??? (jmx.properties)
+ * ??? (recipientrewritetable.xml)
```

Comment:
Configures storage for recipient rewrite rules.

Reply:
Thanks. Would be nice to have a little more in-depth information like [this](https://docs.microsoft.com/en-us/exchange/architecture/edge-transport-servers/address-rewriting?view=exchserver-2019) or [this](https://www.exim.org/exim-html-current/doc/html/spec_html/ch-address_rewriting.html).

Reply:
I wonder if this shouldn't be an "advanced" feature somehow. I would like to be able to simplify as much as possible the "default" server to make it less complex and likely to break for new users.

Reply:
Oh I see.

The idea would be to package 2 JPA based server, one with (almost) no configuration, but close to 0 extra features. Easy to use, you don't have zillion of things to understand. And another one also using JPA but exposing complex configuration to allow customizing it. (the fact of using both JPA would be a coincidence bringed in for technical purposes).

I just got it!

Smart! :+1:

However regarding RRT it enables "alias" which is a very common featurethat even basic users could want (I will run out of time today for your "user email" that is really pertinent). We might not need any configuration for it for "basic users".

Reply:
I see. I will get back to this topic later. First I want to better understand the configuration options, then maybe later we can figure out what we should keep or drop for the "Default" (Basic) Server.